package.json

```diff
@@ -54,7 +54,7 @@
     ]
   },
   "devDependencies": {
-    "@playwright/test": "^1.40.1",
+    "@playwright/test": "1.40.1",
```

Comment:
Do we need this? Does it has to be in sync with ci.yml?

Reply:
Yes. we can write another script to get the playwright version, let me try

Reply:
Never mind, I don't wan't write another script for windows :(

Reply:
```suggestion
    "@playwright/test": "^1.40.1",
```